### DISCUSSION TITLE
Ignoring the notch entirely

### DISCUSSION THREAD

**Discussion by kadrach** [asmvik/yabai#1502] [Q&A] [answered]
Hey all! Is there a way to completely *ignore* the notch on Apple Silicon macbook, and treat the display as a larger rectangle (i.e. think negative top padding)? I've played around with `external_bar` without much luck. 

I've played with a few of the suggestions in https://github.com/koekeishiya/yabai/issues/1060, but this seems to primarily focus on users that _want_ a bar.

**Comment by HeySora** [comment]
@kadrach A simple way to do so is by editing the source code itself:

https://github.com/koekeishiya/yabai/blob/206c0d6968210f48363f206526760e5750810a61/src/workspace.m#L102-L115

This function simply needs to always `return 0;`, like this:

```objc
 int workspace_display_notch_height(uint32_t did) 
 { 
     return 0; 
 } 
 ```

**Comment by asmvik** [comment, marked as answer]
It is not possible to move an arbitrary window on macOS behind the area that is reserved for the notch; I've already experimented with this and confirmed this limitation. It is possible as an application developer to force a window up there, but it cannot be done by a third-party application to arbitrary windows.

I did the above test on Monterey, and I assume that the same applies to Ventura.
